doc/classes/AnimationNodeBlendSpace1D.xml

```diff
@@ -41,13 +55,36 @@
 				Returns the position of the point at index [param point].
 			</description>
 		</method>
+		<method name="get_blend_point_stored_name" qualifiers="const">
+			<return type="String" />
+			<param index="0" name="point" type="int" />
+			<description>
+				Returns the actual stored name of the blend point at index [param point]. Unlike [method get_blend_point_name], this returns an empty string if no name was assigned.
+			</description>
+		</method>
 		<method name="remove_blend_point">
 			<return type="void" />
 			<param index="0" name="point" type="int" />
 			<description>
 				Removes the point at index [param point] from the blend axis.
 			</description>
 		</method>
+		<method name="reorder_blend_point">
+			<return type="void" />
+			<param index="0" name="from_index" type="int" />
+			<param index="1" name="to_index" type="int" />
+			<description>
+				Swaps the blend points at indices [param from_index] and [param to_index], exchanging their positions and properties.
+			</description>
+		</method>
+		<method name="set_blend_point_name">
+			<return type="void" />
+			<param index="0" name="point" type="int" />
+			<param index="1" name="name" type="String" />
+			<description>
+				Sets the name of the blend point at index [param point]. If the name conflicts with existing points, a unique name will be generated automatically.
```

Comment:
```suggestion
				Sets the name of the blend point at index [param point]. If the name conflicts with an existing point, a unique name will be generated automatically.
```
I'd say, as there's only one possible since the names are unique